packages/form/src/token.ts

```diff
@@ -30,7 +30,7 @@ export interface ValidateFieldCallback {
 export interface ElFormItemContext {
   prop?: string
   formItemMitt: Emitter
-  size: ComponentSize
+  size?: string
```

Comment:
We should import `ComponentSize` from `@element-plus/utils/types` instead

Reply:
Okay~

Reply:
@JeremyWuuuuu

Reply:
@imswk Hi, I left a comment, please address that so we can get this merged.